cmd/podman/main_remote_supported.go

```diff
@@ -0,0 +1,57 @@
+// +build remoteclient
+// +build linux darwin
```

Comment:
These should all be on the same line

Reply:
i prefer this notation when mixing os's with something like remote client ... it is perfectly legit.